clowdapp.yaml

```diff
@@ -114,7 +114,7 @@ objects:
           value: "${SETTINGS__FORCE_IMPORT_SSGS}"
         - name: SETTINGS__REPORT_DOWNLOAD_SSL_ONLY
           value: ${SETTINGS__REPORT_DOWNLOAD_SSL_ONLY}
-    - name: compliance-backend
+    - name: service
```

Comment:
This change would also change the Service name, which is mapped on 3scale to route to this application.
We might have an outage when migrating.